File: cpp/estructura_de_datos/data_structures/Queue/java/queue/Node.java
public class Node {

	private int load; // Carga del nodo
	private Node next; // Apuntador al siguiente Nodo

	public Node(int load) {
		this.load = load;
		this.next = null;
	}

	public int getLoad() {
		return load;
	}

	public void setLoad(int load) {
		this.load = load;
	}

	public Node getNext() {
		return next;
	}

	public void setNext(Node next) {
		this.next = next;
	}

}

File: cpp/estructura_de_datos/data_structures/LinkedList/cpp/clases/Node.cpp
#include <iostream>
#include "Node.h"

Node::Node(int n)
{
    load = n;
    next = NULL;
}

void Node::SetLoad(int n)
{
    load = n;
}

int Node::GetLoad()
{
    return load;
}

void Node::SetNext(Node* n)
{
    next = n;
}

Node* Node::GetNext()
{
    return next;
}